charts/cluster-scanner/values.yaml

```
@@ -16,37 +23,59 @@ global:
     # skipVerifyCertificate might be used in on-premise installations.
     skipVerifyCertificate: false
 
-  scannerMode: "local"  # or "multi"
+  # The scannerMode of the Cluster Scanner. Supported values are `local` or
+  # `multi`.
```

```suggestion
  # The scannerMode of the Cluster Scanner. Supported values are `local` or
  # `multi`. Please refer to docs.sysdig.com for further documentation
```

ideally with the specific link when we'll have that